src/components/search/AggregateSearchModal/__tests__/AggregateSearchModal-test.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
+
+jest.autoMockOff();
+
+var React                = require('react/addons');
+var TestUtils            = React.addons.TestUtils;
+var AggregateSearchModal = require('../');
+var findByClass          = TestUtils.findRenderedDOMComponentWithClass;
+var scryByClass          = TestUtils.scryRenderedDOMComponentsWithClass;
+var searchModal;
+var element;
+
+describe('AggregateSearchModal', function() {
+  beforeEach(function() {
+    searchModal = <AggregateSearchModal autoFocus={ false } />;
+    element     = TestUtils.renderIntoDocument(searchModal);
+  });
+
+  it('renders a modal overlay', function() {
+    var overlayElement = findByClass(element, 'Overlay');
+    expect(overlayElement).toBeDefined();
+  });
+
+  it('renders a loading state while fetching results', function() {
+    element.setState({ isSearching: true, results: true });
+    findByClass(element, 'AggregateSearchModal__footer--loading');
```

Comment:
@bradparker I could wrap it in an `expect` but if it cannot find the class the test fails anyway – is your inclination to add it anyway just to be explicit about what's going on?